wlts/collections/collection_manager.py

```diff
@@ -65,43 +65,43 @@ def insert(self, collection_type, collection_info):
             collection_info (dict): The collection information.
         """
         collection = CollectionFactory.make(collection_type, collection_info)
-        self._collections.append(collection)
+        self._collections[f"{collection.name}"] = collection
 
-    def get_collection(self, name):
+    def collection(self, collection_id: str):
         """Return the collection.
 
         Args:
-            name (str): Identifier (name) of an collection.
+            collection_id (str): Identifier (name) of an collection.
 
         Returns:
             collection: A collection available in the server.
+        """
+        if collection_id in self._collections.keys():
+            return self._collections[collection_id]
+
+    def find_collections(self, names: list):
+        """Return list of collections.
+
+        Args:
+            names (list): A list with the collections names.
 
-        Raises:
-            RuntimeError: If the collection not found.
+        Returns:
+            collections: All collection available in the server.
         """
-        try:
-            for collection in self._collections:
-                if name == collection.get_name():
-                    return collection
-        except ValueError:
-            raise RuntimeError(f"Collection {name} not found!")
+        return [self._collections[x] for x in names]
 
     def collection_names(self):
         """Return all available collections.
 
         Returns:
             list: A list with all collections identifier (name) available in the server.
         """
-        collections_names = list()
-
-        for collection in self._collections:
-            collections_names.append(collection.get_name())
-
-        return collections_names
+        if len(self._collections) > 0:
```

Comment:
When collections is empty, the function is not returning an empty list anymore. Is there any problem with that?

Reply:
Must return empty list